__init__.py

```diff
@@ -267,6 +268,7 @@ class Fast64Settings_Properties(bpy.types.PropertyGroup):
         ],
         default="intersect_action_and_scene",
     )
+    prefer_ci_over_rgba: bpy.props.BoolProperty(name="Prefer Color Indexed Over RGBA")
```

Comment:
Please add in a description to this prop explaining how it works and when it applies.

Reply:
Will do

Reply:
```suggestion
    prefer_ci_over_rgba: bpy.props.BoolProperty(name="Prefer CI Over RGBA")
```
also if the better choice is for this to be on, it should be on by default. it's easy to change and save as default blend

Reply:
bump about the default

the description should be clearer that this is about the auto/default format when choosing a texture, not something that overrides what's in the UI

Reply:
sorry to bother you @Dragorn421 but is it good in its current state? if so can i go ahead and merge?